pkg/cmd/server/kubernetes/node_auth.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
 )
 
-func newAuthenticator(clientCAs *x509.CertPool, anonymousConfig restclient.Config, cacheTTL time.Duration, cacheSize int) (authenticator.Request, error) {
+func newAuthenticator(authenticationClient unversionedauthentication.TokenReviewsGetter, clientCAs *x509.CertPool, cacheTTL time.Duration, cacheSize int) (authenticator.Request, error) {
 	authenticators := []oauthenticator.Request{}
```

Comment:
spawn a follow up issue... we have the same issue with an invalid token and a valid client cert falling back to use the client cert